packages/iris-grid/src/sidebar/conditional-formatting/ConditionEditor.tsx

```diff
@@ -141,7 +141,7 @@ function getNumberInputs(
   conditionValue?: string,
   startValue?: string,
   endValue?: string
-) {
+): JSX.Element | null {
```

Comment:
I think `ReactNode` would make more sense here.

Reply:
Looks like `JSX.Element | null` is the same type used by `React.FC` so going to stick with it for now